README.md

```diff
@@ -1 +1,71 @@
-# Awesome-Books-project-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome books: plain JavaScript with objects
+
+Awesome books is an online book store. It's built with HTML, CSS and mostly JS based on a medium-fidelity wireframe given by Microverse. Adding and removing books from the DOM is made in JavaScript functionality. Also, the books are stored in the local storage so they are retreived when the page reloads.
+
+## Live demo https:
+
+
+## Built With
+
+- HTML5
+- CSS3
+- Visual Studio Code
+- Git & Github
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone repo into your machine. For this, I recommend the following steps:
+  - Open a terminal of your choice (e.g. VS Code integrated terminal)
+  - Select the destination folder (On windows, run cd "_Destination folder_")
+- Open the destination folder in your preferred file explorer
+- Open folder "awesome-books"
+- Open index.html
+
+### Prerequisites
+
+Any modern web browser will do.
+
+### Setup
+
+Follow the steps described before.
+
+## Author
+
+
+
+## collaborator1
+
+👤 **George D. Kruah**
+
+- GitHub: [@dfizzkruah93](https://github.com/dfizzkruah93)
+- LinkedIn: [george-d-kruah-2775ab180](https://linkedin.com/in/george-d-kruah-2775ab180)
+- Twitter: [@DKruah](https://github.com/Kruah)
+
+
+
+
+plain-Js-with-Objects
+## 🤝  Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- Microverse program
+- My code reviewers
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+# Awesome-Books-project-
```

Comment:
- [ ] Please the project title is appearing twice, it will be nice to delete this one, as this will make your readme file more presentable.